common/scala/src/main/resources/application.conf

```diff
@@ -167,6 +167,7 @@ whisk {
         min = 100 ms
         max = 5 m
         std = 1 m
+        timeoutfactor = 1
```

Comment:
Can you add a comment as this knob isn’t obvious.

Reply:
Done